### DISCUSSION TITLE
Usage with Courier

### DISCUSSION THREAD

**Discussion by nandorojo** [sofn-xyz/mailing#157] [Q&A] [answered]
Hey there, first off, I'm so glad this exists. Thanks for releasing it.

I want to use Mailing to generate templates to use on [Courier](https://courier.com). Since courier supports MJML for custom templates, would I just run `mailing export-previews` and copy the resulting HTML/MJML into Courier's editor?

Thank you!

**Comment by psugihara** [comment, marked as answer]
Hey @nandorojo, sorry I missed this one. Courier looks cool and yes that should work. We don't have a way to export mjml at this time, but the html you get from export-previews can just be copied in.
